src/main/java/org/jenkinsci/plugins/stashNotifier/NotificationContext.java

```diff
@@ -10,10 +10,12 @@
 public class NotificationContext {
     private final PrintStream logger;
     private final String runId;
+    private final BuildInformation buildInformation;
 
-    public NotificationContext(PrintStream logger, String runId) {
```

Comment:
Can we leave the old constructor too? Happy to mark it deprecated, just would prefer the next release doesn't cause compilation issues if possible.